CHANGES

```diff
@@ -1,6 +1,16 @@
 Injector Change Log
 ===================
 
+0.14.0 (not released yet)
+------
+
+- Fixed a bug with values shared between Injectors in a hierarchy (bugs #52 and #72)
+
+Backwards incompatible:
+
+- If you use a custom scope with child Injectors you need to bind the scope not only with the
+  root Injector but also the children
```

Comment:
I don't think this constraint is ideal. Maybe `bind_scope` can store the bound scope classes, and when a child Binder is constructed it can create new ones. Then this would be completely backwards-compatible.

Reply:
That's a good point, I vaguely thought of something similar as I was writing this. This may need to wait for a day or two before I get a chance to get back to it and implement the change.

Reply:
I had an idea for an even simpler (conceptually) solution - ad hoc scope binding/creation. Let me know what do you think.